sparksteps/__main__.py

```diff
@@ -135,4 +136,10 @@ def main():
                                   args.uploads,
                                   args.s3_dist_cp)
 
-    client.add_job_flow_steps(JobFlowId=cluster_id, Steps=emr_steps)
+    response = client.add_job_flow_steps(JobFlowId=cluster_id, Steps=emr_steps)
+
+    try:
+        step_ids = json.dumps(response["StepIds"])
+    except TypeError:
+        step_ids = "Invalid response"
+    logger.info("Step IDs: %s" % step_ids)
```

Comment:
better practice to write as `logger.info("Step IDs: %s", step_ids)`. this will lazily format the string.